### DISCUSSION TITLE
installed and running yabai, config file loaded but nothing happens

### DISCUSSION THREAD

**Discussion by HosAkh** [asmvik/yabai#1069] [Q&A]
I might be doing something dumb. I installed yabai from `brew install yabai --HEAD`. It installed `yabai-v4.0.0`. 

made a yabai config file and did `chmod`. SIP is disabled and added bunch of things to my config file just to see anychanges and nothing is happening. 

Yabai is running and I gave it access in accessbility. 

**Config file**
```
yabai -m config debug_output on

# bsp or float (default: bsp)
yabai -m config layout bsp

# Set all padding and gaps to 20pt (default: 0)
yabai -m config top_padding    20
yabai -m config bottom_padding 20
yabai -m config left_padding   20
yabai -m config right_padding  20
yabai -m config window_gap     20

# Floating point value between 0 and 1 (default: 0.5)
yabai -m config split_ratio 0.5

# New window spawns to the left if vertical split, or top if horizontal split
yabai -m config window_placement first_child

# set focus follows mouse mode (default: off, options: off, autoraise, autofocus)
yabai -m config focus_follows_mouse autoraise
```

did I do something wrong? I don't know why it's not working. I'm running on a 2019 MBP (intel, of course) running on Monterey

**Comment by guidao** [comment]
maybe "sudo yabai --install-sa; sudo yabai --load-sa"

**Reply by HosAkh** [reply]
> sudo yabai --install-sa; sudo yabai --load-sa

This is a response that I got
```
could not locate Dock.app pid
yabai: scripting-addition failed to inject payload into Dock.app!
```

I uninstalled the scripted and reinstalled it and still got the same error:

**commnads that I run to uninstall and reinstall scripts**
```
sudo yabai --uninstall-sa
sudo yabai --install-sa
sudo yabai --load-sa
sudo yabai --install-sa; sudo yabai --load-sa
```

I have also added this to the very top of my **config** file. 

```
sudo yabai --load-sa
yabai -m signal --add event=dock_did_restart action="sudo yabai --load-sa"
```

I have `SIP` already disabled.

**Comment by HosAkh** [comment]
So I ran `pkill Dock` and then `sudo yabai --load-sa` and it didn't give me an error. 

Then I ran `brew services start yabai` and nohing has happened to my deskop and still feels like as if `yabai` wasn't even installed. My config file is posted in my original post. The only different is that in current version I have script loading at the very top of it. 
```
sudo yabai --load-sa
yabai -m signal --add event=dock_did_restart action="sudo yabai --load-sa"

```

![CleanShot 2021-12-01 at 23 32 22@2x](https://user-images.githubusercontent.com/18110110/144363661-931aafe9-ad06-4538-b461-a45a2c1da08a.jpg)

**Reply by guidao** [reply]
You can try "brew serivces stop yabai; yabai -V", and see if there is any useful information

**Reply by HosAkh** [reply]
I got this response. Still googling to find a solution

```
Error: Unknown command: serivces
yabai: could not acquire lock-file! abort..
```

**Reply by HosAkh** [reply]
https://github.com/koekeishiya/yabai/issues/257#issuecomment-532900034

this fixed my issue and it's now working! It's been 2 months and it's finally working! Thank you for all of your help @guidao

**Reply by gre4test** [reply]
Link is dead, use this: https://web.archive.org/web/20201201121202/https://github.com/koekeishiya/yabai/issues/257
